library/core/src/num/f128.rs

```diff
@@ -627,8 +633,10 @@ impl f128 {
     /// ```
     #[inline]
     #[unstable(feature = "f128", issue = "116909")]
+    // #[rustc_const_unstable(feature = "const_float_methods", issue = "130843")]
+    #[rustc_const_unstable(feature = "f128", issue = "116909")]
```

Comment:
It probably makes sense to use `const_float_methods` for `rustc_const_unstable` here, that way one has to enable both feature gates to const-call this function.